client/src/view/icons/User.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```

Comment:
Instead of creating a folder called icons. There is a file called [icons.js](https://github.com/mxs4321/WASPS-Scheduling-System/blob/master/client/src/view/icons.js). If you put your icon in that file it makes it easier to import multiple icons at a time from other files like [here](https://github.com/mxs4321/WASPS-Scheduling-System/blob/master/client/src/view/AcceptRide.js#L3). 

```js
import { Destination, Phone, Calendar, Vertical } from './icons';
```